src/components/Nav.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Nav = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/calculator',
+      text: 'Calculator',
+    },
+    {
+      id: 3,
+      path: '/quote',
+      text: 'Quote',
+    },
+  ];
+
+  return (
+    <section className="section">
+      <div>
+        <h1>Math Magicians</h1>
+      </div>
+      <nav className="nav">
+        <ul className="menuUl">
+          {links.map((link) => (
+            <li key={link.id}>
+              <NavLink to={link.path} className="links">
+                {link.text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </nav>
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] It would be nice if you could add a padding-left to the style of this component so things get aligned well just as it is on the home page. The screenshot below is a guide as to how things should be aligned for good-looking design and consistency. 👍 

![image](https://user-images.githubusercontent.com/50941074/185602637-dcd7c751-a3d8-4954-a50d-918d3f5aada9.png)

- [ ] Also, it would be nice if you could move the calculator up to align it with the text at the left so your styles match with the wireframe.

![image](https://user-images.githubusercontent.com/50941074/185603040-a57aa889-94a2-4c85-81f8-92e6025f9a4c.png)